src/lib/isMobilePhone.js

```diff
@@ -39,7 +39,7 @@ const phones = {
   'en-KE': /^(\+?254|0)(7|1)\d{8}$/,
   'en-MT': /^(\+?356|0)?(99|79|77|21|27|22|25)[0-9]{6}$/,
   'en-MU': /^(\+?230|0)?\d{8}$/,
-  'en-NG': /^(\+?234|0)?[789]\d{9}$/,
+  'en-NG': /^(\+?234|0)[789][0-1]\d{8}$/,
```

Comment:
BTW, are 7,8,9 given to the specific telcos e.g. Glo, MTN, etc? If that's the case, why is the second one 0/1 controlled, thought it is up to them to manage everything else...

Reply:
I found this in [Wikipedia](https://en.wikipedia.org/wiki/Telephone_numbers_in_Nigeria)

Reply:
@profnandaa Yes, the numbers 7,8,9 and the following 2 digits are given to specific telcos in Nigeria, It is the remaining 7 digits and in some cases, the remaining 6 digits are the ones that are not specific to any telco.

Reply:
Ok, please add extra tests for this tweak. 

Also, if there's a way to generalize as opposed to just lock down to the specific offering, this will be good.